cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/DistributedStreamService.java

```diff
@@ -188,9 +188,11 @@ protected void doShutdown() throws Exception {
   @Override
   protected void runOneIteration() throws Exception {
     LOG.trace("Performing heartbeat publishing in Stream service instance {}", instanceId);
-    ImmutableMap.Builder<String, Long> sizes = ImmutableMap.builder();
-    for (StreamSpecification streamSpec : streamMetaStore.listStreams(Constants.DEFAULT_NAMESPACE)) {
-      sizes.put(streamSpec.getName(), streamWriterSizeCollector.getTotalCollected(streamSpec.getName()));
+    ImmutableMap.Builder<Id.Stream, Long> sizes = ImmutableMap.builder();
+    String namespace = Constants.DEFAULT_NAMESPACE;
```

Comment:
i added a comment. this will be done in the next PR that actually namespaces streams.